### ISSUE TITLE
'node_modules' is not recognized on Windows

### ISSUE THREAD

**Issue by tmswr** [jack-pallot/dogpatch#2]
Great framework, however I cannot get any of the following commands to run on Windows:

- npm run dev

- npm run watch

- npm run hot

- npm run production

Using NPM or Yarn creates the following error:
![image](https://user-images.githubusercontent.com/44615905/47733831-1d702180-dc37-11e8-9bf4-dd05ce1a8144.png)

**Comment by tmswr** [comment]
[2018-10-30T16_58_46_840Z-debug.log](https://github.com/jack-pallot/dogpatch/files/2530715/2018-10-30T16_58_46_840Z-debug.log)

**Comment by jack-pallot** [comment]
It looks like this is fairly common when using Laravel Mix on Windows. There's an issue about it [here](https://github.com/JeffreyWay/laravel-mix/issues/151) with a couple of solutions for amending paths inside package.json, I would give these methods a try first. If not you could also run npm update to get the latest dependancies.

**Comment by tmswr** [comment]
I've tried the following but get the following:
`10% building modules 1/1 modules 0 active`
`Webpack is watching the files... 95% emitting`

It hangs past that. *shrug*

**Comment by jack-pallot** [comment]
I've had this before myself, but I can't remember how I fixed it. From memory I had to remove the node_modules folder and completely restart my machine to get the command prompt to behave again. That could just be coincidence though.

There are a couple solutions posted [here](https://github.com/JeffreyWay/laravel-mix/issues/1126#issuecomment-335158212), let me know if any of them work for you.

**Comment by tmswr** [comment]
Here's what I did to make work for any other Windows user:

Add **cross-env** (don't forget to install it [info](https://www.npmjs.com/package/cross-env)) to the beginning of the dev, watch, hot, and production scripts. 

Example "dev" script with changes:
`"dev": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",`

I then had to modify the webpack.mix.js file by adding `mix.setPublicPath('/')` under the Laravel Mix Config. I hope this helps anyone else wanting to use this!
